molecule/default/prepare.yml

```diff
@@ -1,3 +1,14 @@
 ---
 - name: Import upgrade playbook
   ansible.builtin.import_playbook: upgrade.yml
+
+# This is already installed on the AMIs we use, but not on the Docker
+# images we use for our molecule tests
+- hosts: all
+  name: Ensure that sshd is installed
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: Install openssh-server
+      ansible.builtin.package:
+        name: openssh-server
```

Comment:
I don't understand the point of moving this installation to the prepare playbook.  We must install sshd so that `/etc/ssh/sshd_config` exists for us to modify.  That is why I think the original location of `tasks/main.yml` is a better fit for the installation of `openssh-server`.

Reply:
Then this role needs to be updated to otherwise mention that it's doing that. I would never assume that `openssh-server` is installed from all other documentation in this role. The comment for that particular task even makes it sound like it's only there for testing.

Reply:
Please see commits bad3f96 and 8ab66d3.  Feel free to remove 5bc06c8 and bad3f96 and force push to clean up the history.

Reply:
@jsf9k I'm going to leave the commit history for record keeping. I would still like the OpenSSH installation broken out at some point because it's a weird (albeit necessary right now) fit for this role.